src/answers-umd.js

```diff
@@ -160,6 +160,9 @@ class Answers {
         this.core.storage.delete(StorageKeys.SORT_BYS);
         this.core.filterRegistry.clearAllFilterNodes();
 
+        const queryId = this.core.storage.get(StorageKeys.QUERY_ID);
+        data.set('pop-state-queryId', queryId);
```

Comment:
Would it also be accurate to call this the 'prevous-queryId'?

Reply:
If we are navigating back a page, I thought it may be a bit confusing if 'previous' would mean the query id from the page we are navigating from, or the query id from page we are navigating back to. This is technically storing the current/latest queryId as the next one have not been executed yet, if there is a search on the page we are navigating back/forward to.